### DISCUSSION TITLE
how to get username from github profile page

### DISCUSSION THREAD

**Discussion by vijaypotnuru** [open-sauced-craftwork/browser-extension#32] [General]
![image](https://user-images.githubusercontent.com/110402074/226172407-ec06a0b8-48ac-46c6-b9d2-5ac25f414f5d.png)

**Comment by Rishav1707** [comment]
Use `document.location` to fetch the URL from the current window.

**Reply by vijaypotnuru** [reply]
can you provide any example code

**Comment by thekingslee** [comment]
You could get the Url, split by ("/") and pick the third index on the returned array.

**Reply by vijaypotnuru** [reply]
ok bro
